.github/workflows/labeler.yml

```diff
@@ -14,16 +14,9 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
+
     - name: Add agent-java label
-      uses: actions/github-script@v7
-      with:
-        script: |
-          github.rest.issues.addLabels({
-            issue_number: context.issue.number,
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            labels: ["agent-java"]
-          })
+      run: gh issue edit "${NUMBER}" --add-label "agent-java" --repo "${{ github.repository }}"
```

Comment:
you need the support for the `NUMBER` env variable and the `GH_TOKEN: github.secret` or the one with the GITHUB_TOKEN secret value:

IIUC, it supports GH issues and PRs, so the github context might be different, see https://github.com/elastic/apm-agent-python/blob/0473d9b5592861ed4ca5dea43ecf6b66d4fa35ff/.github/workflows/labeler.yml#L17-L19